portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/includeSitesNav.vm

```diff
@@ -20,8 +20,7 @@
                 #if (!${site.isMyWorkspace} && (${site.favorite} == "true" || ${site.isCurrentSite} || !${userIsLoggedIn}))
                     <li class="Mrphs-sitesNav__menuitem #if (${site.isCurrentSite}) is-selected #end">
                         <a href="${site.siteUrl}" title="${site.fullTitle}" role="menuitem" aria-haspopup="true">
-                            <span>#if ( ( ${tabDisplayLabel} == 2 ) && ( ${site.shortDescription} ) )${site.shortDescription}#else${site.siteTitle}#end</span>
-                            <span class="Mrphs-sitesNav__dropdown" tabindex="-1" data-site-id="${site.siteId}"></span>
+                            <span class="Mrphs-sitesNav__dropdown" tabindex="-1" data-site-id="${site.siteId}>#if ( ( ${tabDisplayLabel} == 2 ) && ( ${site.shortDescription} ) )${site.shortDescription}#else${site.siteTitle}#end"</span>
```

Comment:
I believe the double quote here is incorrectly placed and should be within the <span> tag after site.siteId} not all the way at the end